autoload/wiki.vim

```diff
@@ -172,6 +172,7 @@ function! s:init_buffer_commands() abort " {{{1
   command! -buffer WikiPageDelete         call wiki#page#delete()
   command! -buffer WikiPageRename         call wiki#page#rename()
   command! -buffer WikiPageToc            call wiki#page#create_toc()
+  command! -buffer WikiJournalIndex       call wiki#journal#make_index()
```

Comment:
I've ordered the commands alphabetically. Not important, but I'd appreciate if you could follow the same idea.